src/components/ColorPicker/ColorPickerItem/ColorPickerItem.st.css

```diff
@@ -0,0 +1,55 @@
+:import {
+    -st-from: "../../../common/formatters.st";
+    -st-named: applyUnit, color, fallback, value, applyOpacity, zeroAsTrue;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultItemSize: 24;
+    DefaultBorderRadius: 12;
+    DefaultFocusColor: color-5;
+}
+
+:vars {
+    /*Overrides*/
+    Size: --overridable;
+    Radius: --overridable;
+}
+
+:vars {
+    colorPickerItemSize: applyUnit(fallback(value(Size), value(DefaultItemSize)), px);
+    colorPickerBorderRadius: applyUnit(fallback(zeroAsTrue(value(Radius)), value(DefaultBorderRadius)), px);
+    focusColor: applyOpacity(color(value(DefaultFocusColor)), 0.6);
+}
+
+.root {
+    -st-states: checked;
+}
+
+.radioOuter {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    box-sizing: border-box;
+    margin: 4px;
+    width: value(colorPickerItemSize);
+    height: value(colorPickerItemSize);
+    border-radius: value(colorPickerBorderRadius);
+    transition: box-shadow 0.1s linear;
+    box-shadow: 0px 0px 0px 0px color(value(focusColor));
+}
+
+/* DIRTY HACK START */
+.root:checked .radioOuter, .radioOuter:hover, input:focus ~ span .radioOuter {
```

Comment:
@giladsegal `dirty hack` relates to the last selector `input:focus ~ span .radioOuter` which is used only for the purpose, when a user focuses one of the radio buttons with keyboard.  in that case we should also add shadow to the radio button, maybe you can suggest a better way?

Reply:
how about exposing an additional styable state in `RadioButton` in `wix-ui-core`?

Reply:
What about the selector here?

Reply:
I agree with @giladsegal.
The radio button from `wix-ui-core` has a `focused` state (on its root), we can use that instead of targeting the input element.
i.e. ```.root:focused .radioOuter {...}```

Reply:
done